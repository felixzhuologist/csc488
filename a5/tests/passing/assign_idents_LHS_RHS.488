{
   var A, B, C : Integer
   C := 44
   B := 5
   A := B + 10
   A := A + 24
   write A, newline, B, newline, C
}