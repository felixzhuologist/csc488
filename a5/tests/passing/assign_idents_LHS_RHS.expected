39
5
44